website/blog/2023-10-26-release-1.5.md

```diff
@@ -0,0 +1,219 @@
+---
+title: Release Notes - Podman Desktop 1.5
+description: Podman Desktop 1.5 has been released!
+slug: podman-desktop-release-1.5
+authors: duffy
+tags: [podman-desktop, release, kubernetes, openshift, onboarding]
+hide_table_of_contents: false
+---
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 1.5 Release! 🎉
+
+With this release of Podman Desktop, we're introducing a new onboarding feature that we hope will earn your seal of approval! But wait... there's so much more!
+
+- **Onboarding**: Easy setup and configuration of **Podman** and **Compose**
+- **Podman 4.7.1**: [Podman 4.7.1](https://github.com/containers/podman/releases) is now included in Windows and Mac installers
+- **Command Palette**: Gain easy access to various commands via a new keyboard-driven command palette
+- **Expanded "Summary" tab for Kubernetes pods**: ...
+- **Environment file support**: ...
+- **Enhancements to the Settings area**: ...
+- **Improved user experience for state changes**: ...
+- **Extension API improvements**: A boatload of improvements to the extension API enabling more goodness from Podman Desktop's extensions
```

Comment:
boatload: I see what you did here, and +1 for all future release notes including nautical-based puns.

Reply:
⛵